doc/nrf/known_issues.rst

```diff
@@ -141,9 +147,7 @@ CIA-604: ATv2 cannot be built for the ``thingy91_nrf9160_ns`` build target with
 CIA-661: Asset Tracker v2 application configured for LwM2M cannot be built for the ``nrf9160dk_nrf9160_ns`` build target with modem traces or memfault enabled
```

Comment:
@simensrostad , can you confirm CIA-661 is fixed? The tag for 2.2.0 is not added, and I want to 2x check it is not an overlook. Thanks.

Reply:
Hi, yes. This has been fixed.